Makefile

```diff
@@ -121,7 +121,7 @@ build-image:
 		$(CONCREATE_CMD) \
 		deactivate; \
 	)
-	sudo docker build --force-rm -t $(DEV_IMAGE_FULL_NAME) ./target-docker/image
+	concreate build --target target-docker --tag $(DEV_IMAGE_FULL_NAME)
```

Comment:
I think we also need to update the `Makefile` so that it deletes `target` instead of `target-docker`

Reply:
No, no.. it should be fine as is... I'm using `--target target-docker`, so the output directory should remain the same.

Reply:
I'm officially blind!